writehat/static/js/markdown.js

```diff
@@ -47,6 +47,19 @@ var default_toolbar_icons = [
     className: 'fa fa-sticky-note',
     title: 'Footnote'
   },
+  {
+    name: 'todo',
+    action: function(editor) {
+      var selections = editor.codemirror.getSelections();
+      var highlighted = [];
+      for (var i = 0; i < selections.length; i++) {
+        highlighted.push('{todo|}');
```

Comment:
when inserted without any text after the pipe, the javascript that spawns the notification icon does not appear. This could result in some confusion if someone does not put a description for the todo or just forgets. I would recommend putting a placeholder after the pipe symbol or removing it altogether for now.

![image](https://github.com/blacklanternsecurity/writehat/assets/5758342/0119c7d6-309f-4cd4-aab8-dc1f693d2b86)

![image](https://github.com/blacklanternsecurity/writehat/assets/5758342/01ed6030-cf02-4479-a837-aa2611cd328f)